release/release_gold.sh

```diff
@@ -513,6 +513,20 @@ function tag_release {
 			return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
 		fi
 	done
+
+	if [[ "${_PRODUCT_VERSION}" == 7.4.*-u* ]]
```

Comment:
@davisaints, shouldn't we use quotes in `== 7.4.*-u*` as `== "7.4.*-u*"` to keep consistency with the existing similar statements in our code?

See:
![image](https://github.com/user-attachments/assets/47db6eb6-0db6-4733-a612-7c24b28b0f52)